tests/neg/missing-implicit3.check

```diff
@@ -0,0 +1,18 @@
+-- Error: tests/neg/missing-implicit3.scala:3:37 -----------------------------------------------------------------------
+3 |val sortedFoos = List(new Foo).sorted // error
+  |                                     ^
+  |                  No implicit Ordering defined for B
+  |
+  |                  where:    B is a type variable with constraint >: Foo
+  |                  ..
+  |                  I found:
+  |
+  |                      scala.math.Ordering.ordered[A](/* missing */implicitly[scala.math.Ordering.AsComparable[B]])
```

Comment:
Could you use some locally defined implicits instead of depending on what is in the standard library? It makes the test more stable in case the library changes, and it seems that even when the library don't change our output is not stable based on earlier failures of this PR (this is something we should try to fix, but it might be hard).

Reply:
Sure I can do that. On the other hand, please be aware that by creating artificial test scenarios we might simplify the reality and miss bugs/limitations.

This is what motivated this PR: the existing test suite for import suggestions was only based on artificial scenarios and we didn’t know that the output of the compiler was not good on some real world cases.

Reply:
Fixed and squashed.